frontend/src/components/IssueV1/components/HeaderSection/Assignee/Assignee.vue

```diff
@@ -134,11 +143,145 @@ const changeAssigneeUID = async (uid: string | undefined) => {
 };
 
 const filterAssignee = (user: User): boolean => {
-  return (
-    assigneeCandidates.value.findIndex(
-      (candidate) => candidate.name === user.name
-    ) >= 0
+  return isMemberOfProjectV1(issue.value.projectEntity.iamPolicy, user);
+};
+
+const mapUserOptions = (users: User[]) => {
+  const project = issue.value.projectEntity;
+
+  // Project owners go top
```

Comment:
We can have reviewer/releaser of the current step/stage to go top, then project owners, workspace owner/DBA and others.